public/components/query_compare/search_result/result_components/result_panel.tsx

```diff
@@ -47,7 +56,9 @@ export const ResultPanel = ({ resultNumber, queryResult }: ResultPanelProps) =>
           </EuiTitle>
         </EuiFlexItem>
       </EuiFlexGroup>
-      {queryResult?.hits?.hits?.length ? (
+      {queryError.errorResponse.statusCode !== 200 || queryError.queryString.length ? (
+        ErrorMessage
```

Comment:
should it be `<ErrorMessage />`?

Reply:
Also this only have the message, do we consider reveal the status code?

Reply:
Including the status code would be helpful too. Do you have any suggestions on where to include it? So far, I'm thinking:
```
Query Error: <status  code>
<error message>
```
or
```
Query Error
<error message>
Status: <status code>
```

Reply:
How about `<status  code> <error type> - <error message> `, if the status code is 5xx, should it still a query error?